Shared/ThresholdKeyView.swift

```diff
@@ -309,13 +347,17 @@ struct ThresholdKeyView: View {
                     HStack {
                         Text("Add password")
                         Spacer()
+                        if isLoadingAddPassword {
+                            LoaderView()
+                        }
                         Button(action: {
                             // TODO: allow users to input password in a popup.
                             // TODO: add loader as well, API call could take a >3 seconds
                             let question = "what's your password?"
                             let answer = "blublu"
```

Comment:
We should prompt the user for the initial password here as well, like you have done for change password.

Reply:
IMHO, I like the experience where dev has to click on add password btn and without friction password gets added, And  can be changed with change password.

Reply:
And if the user elects not to/forgets to change it? Then the password for the share will be public knowledge.

Reply:
Generated a random password, please check randomPassword func